lib/active_record_doctor/schema_cache.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+module ActiveRecordDoctor
+  class SchemaCache # :nodoc:
+    attr_reader :connection
+
+    def initialize(connection)
+      @connection = connection
+      @indexes = {}
+      @foreign_keys = {}
+      @check_constraints = {}
+    end
+
+    def tables
```

Comment:
Seems these methods were simply moved without changes (except for caching), but @fatkodima please confirm.

Reply:
Yes.